packages/teams-components/src/components/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  type ToggleButtonProps as ToggleButtonPropsBase,
+  useToggleButtonStyles_unstable,
+  useToggleButton_unstable,
+  renderToggleButton_unstable,
+  Tooltip,
+} from '@fluentui/react-components';
+import { validateStrictClasses, type StrictCssClass } from '../../strictStyles';
+import type { StrictSlot } from '../../strictSlot';
+import { validateIconButton } from '../Button/validateIconButton';
+
+export interface ToggleButtonProps
```

Comment:
Should it reuse `ButtonProps`?

Reply:
Yes I believe some parts of this should be shared, let me check

Reply:
Yeah just making everything extend the strict button props makes much more sense